.babelrc

```diff
@@ -1,3 +1,3 @@
 {
-  "presets": ["react", "es2015"]
+  "presets": ["react", "es2015", "stage-0"],
```

Comment:
It need to for this.

`static propTypes = { ... }`

I did change 'stage-0' preset to 'transform-class-properties' plugins.

Good catch! Thank you!